ugit

```diff
@@ -306,6 +307,19 @@ undo_git_tag() {
     fi
 }
 
+undo_git_rm(){
+    # Find all the deleted files from git history
+    filepath=$(git log --diff-filter=D --summary | grep delete | nl -n ln | fzf --header="Choose the file to restore" --height 50% --ansi --reverse --pointer='👉' --cycle | awk '{print $5}')
+    # Find the commit hash value of the selected filepath
+    commit=$(git log -- "$filepath"| awk 'NR==1{print $2}') 
+
+    # Restore the file
+    if git checkout "$commit"^ "$filepath"; then
+        printf "%s\n" "File $filepath restored successfully." 
+        printf "%s\n" "Make sure to run ${BOLD_ORG_FG}git push${RESET} now!"
```

Comment:
```suggestion

```
Let's remove this, someone might `add` & `commit` those changes. Who knows they might want to edit it. Saying that they should "push" the file doesn't really help the user.